sample/bucketclass.yaml

```diff
@@ -0,0 +1,13 @@
+kind: BucketClass
```

Comment:
I would think that all these sample yamls would reside somewhere else in cosi, maybe the central controller repo, or spec, or general doc?

Reply:
I can open a separate PR for these in another repository. What do you think @sidharthsurana @brahmaroutu?